src/style.css

```
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+}
+
+:root {
+  --star-active: url("./images/star-active.svg") center no-repeat;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars div:not(:first-child) {
```

style classes instead of tag ones
```suggestion
.stars_star:not(:first-child) {
```